pkgs/applications/science/astronomy/siril/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, fetchFromGitLab, fetchFromGitHub, pkg-config, meson, ninja,
+  git, criterion, wrapGAppsHook, gtk3, libconfig, gnuplot, opencv,
+  fftwFloat, cfitsio, gsl, exiv2, curl, librtprocess, ffmpeg,
+
+  # Optional deps
+  libraw ? null, libtiff ? null, libpng ? null, libjpeg ? null,
```

Comment:
Please order the libs after there option like 

```
rawSupport ? true, libraw,
```

Is there a use case where you wouldn't want support with all libs? Maybe just enable them by default.

Reply:
Use case is there but weak. Someone may want a slimmed down build that only includes dependencies used in their camera & astro-processing workflow (since they don't tend to change quickly over time). Latest commit solves the optional dep ordering issue so i'm marking this resolved but I can drop the optional logic and make only fully featured builds supported if you think that's the cleaner way to go.

Reply:
It would make the package a lot simpler and all of them are turned on by default anyway.